google/resource_compute_instance_test.go

```diff
@@ -1331,6 +1380,34 @@ func testAccComputeInstance_local_ssd(instance string) string {
 	}`, instance)
 }
 
+func testAccComputeInstance_scratchDisk(instance string) string {
+	return fmt.Sprintf(`
+	resource "google_compute_instance" "scratch" {
```

Comment:
Same nit as https://github.com/terraform-providers/terraform-provider-google/pull/122#discussion_r122062396 and https://github.com/terraform-providers/terraform-provider-google/pull/122#discussion_r122062678;